cis/gcp/compute/gcp-cis-4.2-compute-block-project-wide-ssh-keys-enabled-for-vm-instances/gcp-cis-4.2-compute-block-project-wide-ssh-keys-enabled-for-vm-instances.sentinel

```diff
@@ -19,7 +19,7 @@ print("CIS 4.2: Ensure 'Block Project-wide SSH keys' are enabled for VM instance
 
 deny_undefined_compute_instance_metadata = rule {
 	all allComputeInstances as _, instances {
-		keys(instances.change.after) contains "metadata"
+		keys(instances.change.after) contains "metadata" and instances.change.after.metadata is not null
```

Comment:
```suggestion
		instances.change.after.metadata is not null
```
@luckoseabraham I have generated a terraform plan and it appears that the GCP provider will assign a null value to the metadata attribute when it is not configured within a Terraform template and therefore we can probably drop the key lookup from within the metadata map.

Thoughts?

Reply:
@hcrhall  - Sounds good to me ! 
I did the change because I thought you might want to retain the original plan with metadata attribute not present. 
Will make the change 👍